### PR TITLE
fix: files moved

### PR DESCRIPTION
**Pull request by mayuran-deriv**

### CODE REVIEW

**Review by hasan-deriv — CHANGES_REQUESTED** [review]
Fonts should be added in public directory.

**Review by matin-deriv — CHANGES_REQUESTED** [review]
I believe majority of the files are not needed and all the static files need to be moved to the public folder.

You can check out this PR for the unnecessary files: https://github.com/deriv-com/binary-static/pull/558

File: public/images/pages/header/ic-brand-dbot.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="35" height="35"><g fill="none" fill-rule="evenodd"><rect width="35" height="35" fill="#FF6444" fill-rule="nonzero" rx="6"/><path d="M13.72 13.26a.845.845 0 100-1.69.845.845 0 000 1.69z"/><path fill="#FFF" fill-rule="nonzero" d="M25.26 15.43v-4.6A4.84 4.84 0 0020.43 6h-3.86v9.43h8.69zM16.57 26h3.86a4.84 4.84 0 004.83-4.83v-4.6h-8.69V26zm-4.775-12.1a2.825 2.825 0 112.77-3.36h.865V6h-3.86a4.84 4.84 0 00-4.83 4.83v10.34A4.84 4.84 0 0011.57 26h3.86V11.605h-.865a2.825 2.825 0 01-2.77 2.295z"/><path d="M0 0h32v32H0z"/></g></svg>
```
Comment (henry-deriv) [inline public/images/pages/header/ic-brand-dbot.svg:1]: - ic-brand-binarybot
- ic-brand-dbot
- ic-brand-dtrader

are old logos. Needs to be rebranded logos

File: public/images/pages/footer/linkedin.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60"><path d="M46.837 44.137V32.425c0-6.275-3.35-9.195-7.816-9.195-3.603 0-5.218 1.983-6.118 3.374V23.71h-6.79c.09 1.917 0 20.427 0 20.427h6.79V32.73c0-.61.044-1.22.224-1.656.49-1.22 1.607-2.483 3.482-2.483 2.458 0 3.44 1.874 3.44 4.62v10.928h6.79zM18.96 20.922c2.366 0 3.84-1.57 3.84-3.53-.043-2.004-1.474-3.53-3.796-3.53s-3.84 1.525-3.84 3.53c0 1.96 1.473 3.53 3.752 3.53h.043zM30 60C13.432 60 0 46.568 0 30 0 13.43 13.432 0 30 0c16.568 0 30 13.43 30 30 0 16.568-13.432 30-30 30zm-7.646-15.863V23.71h-6.79v20.427h6.79z" fill="#dcdcdc" fill-rule="evenodd"/></svg>
```
Comment (henry-deriv) [inline public/images/pages/footer/linkedin.svg]: I've never seen the following social media icons in the footer. Also, we'll try to match app.deriv.com footer so u'll have to find those icons as well.

- linkedin
- facebook
- reddit
- telegram
- twitter
- youtube
need to be removed

**Comment by henry-deriv** [comment]
Screenshot 2023-10-11 at 9 00 14 AM


`trade-explanation` svgs are missing. There should be one for each supported language.

File: public/images/pages/header/logo_smart_trader.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg width="33" height="33" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M6.07 0h20.24a6.07 6.07 0 016.07 6.07v20.24a6.07 6.07 0 01-6.07 6.07H6.07A6.07 6.07 0 010 26.31V6.07A6.07 6.07 0 016.07 0" fill="#2A3052"/><path d="M19.796 13.78l-4.444-2.012.736 3.704 2.387 1.095a3.492 3.492 0 01-2.349 6.559 3.086 3.086 0 01-.524-.178l-2.27-1.044-3.491 1.714 4.546 2.067a6.543 6.543 0 005.409-11.905z" fill="#FFF" fill-rule="nonzero"/><path d="M13.923 15.894a3.492 3.492 0 012.35-6.561c.178.045.354.104.523.178l2.292 1.04 3.492-1.713-4.543-2.067a6.682 6.682 0 00-1.047-.359 6.549 6.549 0 00-4.372 12.27l4.445 2.012-.756-3.73-2.384-1.07z" fill="#FFF" fill-rule="nonzero"/></g></svg>
```
Comment (henry-deriv) [inline public/images/pages/header/logo_smart_trader.svg:1]: needs to be rebranded logo

File: public/images/pages/settings/2fa.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg width="126" height="126" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M2 124h122V2H2v122zM0 0h126v126H0V0z" fill="#CDCCCB" fill-rule="nonzero"/><path fill="#F2F2F2" d="M2 2h122v122H2z"/><g fill="#2A3052" fill-rule="nonzero"><path d="M78 45.14a3 3 0 0 0-.756.364L69 51v-9H48v42h30V45.14zm3-.14v43.5a4.5 4.5 0 0 1-4.5 4.5h-27a4.5 4.5 0 0 1-4.5-4.5v-51a4.5 4.5 0 0 1 4.5-4.5H69v-6a3 3 0 0 1 3-3h27a3 3 0 0 1 3 3v15a3 3 0 0 1-3 3H81zm-12-9H49.5a1.5 1.5 0 0 0-1.5 1.5V39h21v-3zm9 51H48v1.5a1.5 1.5 0 0 0 1.5 1.5h27a1.5 1.5 0 0 0 1.5-1.5V87zm-6-41.606l3.58-2.386A6 6 0 0 1 78.908 42H99V27H72v18.394zM76.5 36a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm6 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm6 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm6 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"/><path d="M57 63v9h12v-9H57zm12-3a3 3 0 0 1 3 3v9a3 3 0 0 1-3 3H57a3 3 0 0 1-3-3v-9a3 3 0 0 1 3-3v-3a6 6 0 1 1 12 0v3zm-9 0h6v-3a3 3 0 0 0-6 0v3z"/></g></g></svg>
```
Comment (henry-deriv) [inline public/images/pages/settings/2fa.svg]: We don't have settings. So the whole folder can be removed.

File: public/images/social/apple.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path d="M17.294 12.473c0-2.46 2.063-3.65 2.142-3.73-1.19-1.665-2.935-1.903-3.57-1.983-1.508-.158-3.015.873-3.73.873-.793 0-1.983-.873-3.252-.873-1.666 0-3.174.952-4.046 2.46-1.746 3.015-.476 7.458 1.19 9.917.873 1.19 1.825 2.539 3.094 2.46 1.27-.08 1.745-.794 3.253-.794 1.507 0 1.904.794 3.253.794 1.349 0 2.221-1.19 3.015-2.38.952-1.35 1.348-2.698 1.348-2.777-.079-.08-2.697-1.032-2.697-3.967zm-2.46-7.299C15.47 4.3 15.945 3.19 15.866 2c-.952.08-2.222.635-2.856 1.507-.635.714-1.19 1.905-1.032 3.015 1.032.08 2.142-.555 2.856-1.348z" fill="#000" fill-rule="nonzero"/></svg>
```
Comment (henry-deriv) [inline public/images/social/apple.svg]: no need for social icons

File: public/images/common/email_sent.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg width="152" height="58" viewBox="0 0 152 58" xmlns="http://www.w3.org/2000/svg"><title>message</title><g fill-rule="nonzero" fill="none"><path d="M151.932 4.313a4.315 4.315 0 0 0-4.314-4.299h-.006l-87.303.12A4.315 4.315 0 0 0 56 4.444l.068 49.23a4.315 4.315 0 0 0 4.314 4.3h.006l87.303-.12a4.29 4.29 0 0 0 3.05-1.265A4.272 4.272 0 0 0 152 53.543l-.068-49.23zm-9.056.013l-38.872 26.966-38.95-26.86 77.822-.106zM120.21 35.612l23.185 17.943-78.853.108 23.322-18.05a2.15 2.15 0 0 0 .383-3.021 2.16 2.16 0 0 0-3.027-.382L60.38 51.436 60.316 6.4l42.464 29.282a2.157 2.157 0 0 0 2.458-.004l42.382-29.4.064 45.15-24.83-19.218a2.16 2.16 0 0 0-3.027.382 2.15 2.15 0 0 0 .382 3.02z" fill="#2A3052"/><path d="M2.5 17.5a2 2 0 1 1 0-4h40a2 2 0 1 1 0 4h-40zM11 30.5a2 2 0 1 1 0-4h31.5a2 2 0 1 1 0 4H11zM21 43.5a2 2 0 1 1 0-4h21.5a2 2 0 1 1 0 4H21z" fill="#E98024"/></g></svg>
```
Comment (henry-deriv) [inline public/images/common/email_sent.svg]: wheres this used?

File: public/images/common/globe-with_logo.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...22.46M677.7 73.41q275.7 356 0 712M684.43 78.75q282.41 350.67 0 701.36M691.16 84.21q289.14 345.23 0 690.44M697.88 89.8q295.86 339.63 0 679.25M704.61 95.52q302.58 333.92 0 667.81M711.33 101.37q309.3 328.06 0 656.12M718.05 107.35q316 322.08 0 644.16M724.77 113.45q322.76 316 0 632M731.5 119.69q329.49 309.74 0 619.48"/></g><path d="M261.85 644.57v.06l21.26 27.64-34 27.64v.06h34.37a21.54 21.54 0 0021-16.69l2.79-12.28a21.54 21.54 0 00-21-26.38z" fill="url(#linear-gradient)"/><path fill="#cc2e3d" d="M283.08 672.33l-14.94 27.64h-19.08l34.02-27.64z"/><path class="cls-7" d="M334.59 644.57h21.6c17.42 0 27.34 11.95 24.59 27.54v.16C378 687.86 363.84 700 346.42 700h-21.6zM348.36 689A19.53 19.53 0 00368 672.43v-.16c1.73-9.81-3.78-16.7-13.76-16.7h-9.41L339 689zM396.83 644.57H439l-1.91 10.84H407l-2 11.24h19.2l-1.91 10.85h-19.16l-2.05 11.63h30.08L429.25 700h-42.19zM478.34 680.19c7.51-2.62 13.33-8.24 14.91-17.18v-.16c.92-5.22 0-9.58-2.51-12.66-3-3.64-8.11-5.62-15.16-5.62h-25.31L440.5 700h12.19l7.83-44.4h12.11c5.94 0 9.11 2.69 8.18 7.92v.15c-.83 4.67-4.79 7.84-10.73 7.84h-9.18L474.14 700h14.25zM507.72 644.57h12.19L510.14 700H498zM529.76 644.57h13.36l7.56 38.74 21.21-38.74h13.05l-32 55.4h-10.7z"/></svg>
```
Comment (henry-deriv) [inline public/images/common/globe-with_logo.svg:1]: dont think this is used

**Comment by henry-deriv** [comment]
`.png` files can be removed.

File: public/images/common/password_hide.svg [hunk 8]
```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14.177 2.118a.5.5 0 0 1 .646.764l-2.048 1.732A7.924 7.924 0 0 1 15.333 8c-1.153 2.927-4 5-7.333 5a7.83 7.83 0 0 1-3.908-1.038l-2.269 1.92a.5.5 0 1 1-.646-.764l2.048-1.733A7.925 7.925 0 0 1 .667 8C1.82 5.073 4.667 3 8 3a7.83 7.83 0 0 1 3.909 1.038l2.268-1.92zm-2.265 3.226L10.58 6.472c.267.448.42.97.42 1.528 0 1.653-1.347 3-3 3-.736 0-1.41-.267-1.933-.708l-1.106.935c.918.5 1.953.773 3.04.773 2.584 0 4.877-1.55 6-4a7.047 7.047 0 0 0-2.089-2.656zM8 4C5.422 4 3.123 5.55 2 8a7.05 7.05 0 0 0 2.087 2.656L5.42 9.528A2.98 2.98 0 0 1 5 8c0-1.653 1.347-3 3-3 .736 0 1.411.267 1.934.709l1.105-.935A6.336 6.336 0 0 0 8 4zm1.802 3.13L6.845 9.633A2 2 0 0 0 9.802 7.13zM8 6A2 2 0 0 0 6.2 8.87l2.956-2.503A1.99 1.99 0 0 0 8 6z" fill="#333"/></svg>
```
Comment (henry-deriv) [inline public/images/common/password_hide.svg]: password_show
password_hide

not needed

File: public/images/pages/contact/banner-bg.svg [hunk 9]
```diff
@@ -0,0 +1 @@
...93L47.357-36.15l-1.47 5.127 48.704 13.966 1.47-5.127-20.677-19.8z" fill-opacity=".03" fill="#2A3052" fill-rule="nonzero"/></g><g><path d="M352.426 122.893l-47.533 6.68 6.68 47.534 47.534-6.68z"/><path d="M313.372 132.421l31.689-4.453a4.012 4.012 0 0 1 4.518 3.404l5.01 35.65-9.036-6.809-27.727 3.897a4.012 4.012 0 0 1-4.518-3.404l-3.34-23.767a4.012 4.012 0 0 1 3.404-4.518zm3.897 27.728l27.728-3.897 4.517 3.404-3.896-27.727-31.69 4.453 3.341 23.767z" fill-opacity=".03" fill="#2A3052" fill-rule="nonzero"/></g><g><path d="M295 27.64L356.52 10l17.642 61.52-61.521 17.642z"/><path d="M353.994 49.562l2.94 10.254-5.126 1.47-2.94-10.254 5.126-1.47zm-30.76 8.82l2.94 10.254-2.563.735a2.675 2.675 0 0 1-3.299-1.828l-2.205-7.69 5.127-1.47zm3.261-36.998c-12.74 3.653-20.108 16.945-16.455 29.685l5.146 17.944a7.99 7.99 0 0 0 9.895 5.485l7.69-2.205-5.88-20.507-10.254 2.94-1.47-5.127c-2.844-9.92 2.878-20.244 12.798-23.089 9.92-2.844 20.245 2.879 23.09 12.799l1.47 5.127-10.254 2.94 5.88 20.507 10.254-2.94.735 2.563-17.944 5.145 1.47 5.127 15.38-4.41a7.99 7.99 0 0 0 5.485-9.896l-7.35-25.633c-3.653-12.74-16.946-20.108-29.686-16.455z" fill-opacity=".03" fill="#2A3052" fill-rule="nonzero"/></g></g></g></svg>
```
Comment (henry-deriv) [inline public/images/pages/contact/banner-bg.svg]: dont think this is used

File: public/images/common/globe-with_logo.svg (hunk 7 above)
Comment (mayuran-deriv) [reply public/images/common/globe-with_logo.svg:1]: we need this I saw this in the current project

File: public/images/pages/contact/content-icon.svg [hunk 10]
```diff
@@ -0,0 +1 @@
+<svg width="64" height="64" xmlns="http://www.w3.org/2000/svg"><path d="M50 4a6 6 0 016 6v44a6 6 0 01-6 6H14a6 6 0 01-6-6V10a6 6 0 016-6zm0 4H14a2 2 0 00-2 2v44a2 2 0 002 2h36a2 2 0 002-2V10a2 2 0 00-2-2zm-8 32a2 2 0 110 4H30a2 2 0 110-4h12zm-20 0a2 2 0 110 4 2 2 0 010-4zm20-8a2 2 0 110 4H30a2 2 0 110-4h12zm-20 0a2 2 0 110 4 2 2 0 010-4zm20-8a2 2 0 110 4H30a2 2 0 110-4h12zm-20 0a2 2 0 110 4 2 2 0 010-4zm25.2-12a.8.8 0 01.8.8v5.269a.8.8 0 01-1.366.565l-5.268-5.268A.8.8 0 0141.93 12h5.27z" fill="#2A3052"/></svg>
```
Comment (henry-deriv) [inline public/images/pages/contact/content-icon.svg:1]: where is this used

File: public/images/pages/contact/ticket-icon.svg [hunk 11]
```diff
@@ -0,0 +1 @@
...0 1 0 0-2h-5a1 1 0 1 0 0 2zM30 25.997a1 1 0 0 0 1 1h7a1 1 0 1 0 0-2h-7a1 1 0 0 0-1 1zM27 26.997c.26 0 .52-.11.71-.29.18-.19.29-.45.29-.71 0-.271-.11-.521-.29-.71-.37-.37-1.05-.37-1.42 0-.181.18-.29.439-.29.71 0 .26.109.52.29.71.189.18.45.29.71.29zM32 32.997H22a1 1 0 1 0 0 2h10a1 1 0 1 0 0-2zM18 34.997c.27 0 .52-.11.71-.29.18-.19.29-.45.29-.71 0-.26-.11-.521-.29-.71-.37-.37-1.05-.37-1.42 0-.19.189-.29.45-.29.71 0 .26.1.52.29.71.189.18.45.29.71.29zM42 26.997c.26 0 .52-.11.71-.29.18-.19.29-.45.29-.71 0-.271-.11-.521-.29-.71-.38-.37-1.05-.37-1.42 0-.181.189-.29.439-.29.71 0 .26.1.52.29.71.189.18.45.29.71.29z"/><path d="M54 20.975V8.997H41.135l-8.593-8a3.745 3.745 0 0 0-5.085.001l-8.592 7.999H6v11.978L0 26.56v33.437h60V26.56l-6-5.585zM20.047 42.766c-.019.077-.047.15-.047.231 0 .081.028.155.047.231L2 56.978V29.017l18.047 13.749zm2.291 1.231h15.324l18.375 14H3.963l18.375-14zM58 29.017v27.961l-18.35-13.98L58 29.017zm-.538-2.086c-.031.02-.065.031-.094.053L54 29.549v-5.842l3.462 3.224zM28.819 2.462a1.742 1.742 0 0 1 2.36-.001l7.02 6.536H21.801l7.018-6.535zM8 10.997h44v20.075L37.662 41.997H22.338L8 31.074V10.997zM6 29.55l-3.368-2.566c-.029-.022-.064-.033-.094-.053L6 23.707v5.843z"/></g></svg>
```
Comment (henry-deriv) [inline public/images/pages/contact/ticket-icon.svg]: where is this used

**Comment by henry-deriv** [comment]
Screenshot 2023-10-13 at 2 29 23 PM

lets try to get the missing ones. Except sun svg icon for changing theme

File: public/images/social/facebook.svg [hunk 12]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><rect fill="#3B5998" width="24" height="24" rx="2"/><path d="M12.877 24h3.752v-9.995h2.814L20 10.882h-3.371V8.619c0-.73.486-1.496 1.181-1.496h1.916V4h-2.348v.014c-3.674.13-4.429 2.193-4.494 4.36h-.007v2.508H11v3.123h1.877V24z" fill="#FFF"/></g></svg>
```
Comment (matin-deriv) [inline public/images/social/facebook.svg]: Where do we use social icons?

File: public/images/pages/header/ic-mt5-advanced.svg [hunk 13]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64"><g fill="none" fill-rule="evenodd"><circle cx="32" cy="32" r="32" fill="#C7E5E5" fill-rule="nonzero"/><path fill="#84ABAE" fill-rule="nonzero" d="M37.2 29.76a3.72 3.72 0 016.32 0l4 6.68 4 6.72c.013.12.013.24 0 .36a3.4 3.4 0 01-4.96 4l-4-2.32a4.8 4.8 0 00-4.64 0M27.24 29.72a4 4 0 00-6.56 0l-4 6.72-4 6.72v.32c-1.36 2.88 2.24 5.6 5.12 4l4.2-2.32a5.08 5.08 0 014.84 0"/><path fill="#84ABAE" fill-rule="nonzero" d="M33.36 8.36A2.92 2.92 0 0032 8a2.88 2.88 0 00-1.32.36C26.72 10.24 20 19.32 20 30.28A32 32 0 0024.72 48h14.56A31.56 31.56 0 0044 30.28c0-10.96-6.68-20.04-10.64-21.92z"/><ellipse cx="32" cy="23.76" fill="#C7E5E5" fill-rule="nonzero" rx="4" ry="4.12"/><path fill="#84ABAE" fill-rule="nonzero" d="M30.72 55.12L28 49.4a1.52 1.52 0 01.24-1.64l2.56-3.24a1.4 1.4 0 012.16 0l2.56 3.24a1.52 1.52 0 01.24 1.64l-2.6 5.72a1.36 1.36 0 01-2.44 0z"/><path d="M0 0h64v64H0z"/></g></svg>
```
Comment (matin-deriv) [inline public/images/pages/header/ic-mt5-advanced.svg]: Certain icons such as MT5 ones are not needed! you can remove them 🙏🏼

**Review by matin-deriv — CHANGES_REQUESTED** [review]

File: public/images/social/apple.svg (hunk 5 above)
Comment (mayuran-deriv) [reply public/images/social/apple.svg]: @matin-deriv i removed social icons

Comment (mayuran-deriv) [reply public/images/social/apple.svg]: added again

File: public/images/common/ic_dtrader.svg [hunk 14]
```diff
@@ -0,0 +1,7 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (henry-deriv) [inline public/images/common/ic_dtrader.svg]: rebranded icons need to be minified

**Review by henry-deriv — APPROVED** [review]
LGTM

**Review by matin-deriv — APPROVED** [review]

**Pull request merged**